docs/getting_started.md

```diff
@@ -95,6 +99,25 @@ spec:
 
 Both labels and annotations can be added to the Prometheus configuration, so that metrics are associated with the appropriate information, such as the test and tenant ids. You can read more about this in the [Prometheus documentation](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#kubernetes_sd_config) site.
 
+##### Image pull policy and secrets
```

Comment:
I propose to change the name of this section to `Usage of a private image registry` and moving it to "Advanced topics" (thoughts are welcome). 
Although the implementation under the hood is pull policy and secrets, the main use case here for this implementation is private image registry that requires authentication. 

Since this PR also adds Pull policy and at the same time  you don't want to combine that under the suggested title, I am okay making a new sub-section taking about how to set a pull policy.